src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+
 public class FileWork {
+    private static final String TARGET_CHARACTER = "w";
+    private static final String REGEX_CONST = "\\W";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder();
+        List<String> wordsArray = new ArrayList<>();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
```

Comment:
```suggestion
        try (BufferedReader reader = new BufferedReader(new FileReader(fileName)));
```
This way resources will be closed automatically